CHANGELOG.md

```diff
@@ -17,4 +19,5 @@ All notable changes to this project will be documented in this file.
 - Initial release
 
 [Unreleased]: https://github.com/paralus/helm-charts/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/paralus/helm-charts/releases/tag/v0.1.0
+[0.1.0]: https://github.com/paralus/helm-charts/releases/tag/ztka-0.1.0
+[0.1.1]: https://github.com/paralus/helm-charts/releases/tag/ztka-0.1.1
```

Comment:
This would be:
```suggestion
[Unreleased]: https://github.com/paralus/helm-charts/compare/ztka-0.1.1...HEAD
[0.1.1]: https://github.com/paralus/helm-charts/compare/ztka-0.1.0...ztka-0.1.1
[0.1.0]: https://github.com/paralus/helm-charts/releases/tag/ztka-0.1.0
```

ref: https://keepachangelog.com/en/1.0.0/